Src/Support/Google.Apis.Auth/OAuth2/GoogleCredential.cs

```diff
@@ -83,10 +95,13 @@ internal GoogleCredential(ICredential credential)
         /// </list>
         /// </summary>
         /// <returns>A task which completes with the application default credentials.</returns>
-        public static Task<GoogleCredential> GetApplicationDefaultAsync()
-        {
-            return defaultCredentialProvider.GetDefaultCredentialAsync();
-        }
+        public static async Task<GoogleCredential> GetApplicationDefaultAsync(IEnumerable<string> scopes) =>
```

Comment:
Hmm, true, although this is already the case if you call CreateScoped() manually. I agree this makes it easier for a user to create many different GoogleCredential instances.
This code could cache by scopes, but that's probably more than we want to do.